opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -152,3 +154,57 @@ def _python_path_without_directory(python_path, directory, path_separator):
         "",
         python_path,
     )
+
+
+_OTEL_SEMCONV_STABILITY_OPT_IN_KEY = "OTEL_SEMCONV_STABILITY_OPT_IN"
```

Comment:
```suggestion
_OTEL_SEMCONV_STABILITY_OPT_IN_KEY = "OTEL_SEMCONV_STABILITY_OPT_IN_KEY"
```

Reply:
This existing env var is in line with the [Migration Plan](https://github.com/open-telemetry/semantic-conventions/tree/main/docs/http#semantic-conventions-for-http)

Is this out of date?